README.md

```diff
@@ -102,7 +102,7 @@ the JS bundle. We've chosen to let Rails handle CSS, SCSS, images, fonts.
 
 ```
 cd client
-$(npm bin)/webpack -w --config webpack.rails.config.js
+npm run build:dev
 ```
 
 `client-bundle.js` is generated and saved to `app/assets/javascripts`. This is included in the
```

Comment:
Also when we upgrade to npm@3.3.4 we need to change the npm install/upgrade instructions in the client/REAMDE. `shrinkwrap.json` gets automatically updated during npm install/upgrade similar to Gemfile.loc.

![image](https://cloud.githubusercontent.com/assets/8039859/9981889/c1d0a724-5f65-11e5-9298-1bd69bd94127.png)